### PR TITLE
Update title and blogger name in index.html

### PR DESCRIPTION
**Pull request by abhishekp2917**
Change page tile from My Blog to Abhishek Pandey Blog

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,16 +6,16 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="index.css">
     <link rel="stylesheet" href="bootstrap.css">
-    <title>Abhishek Blog</title>
+    <title>Abhishek Blogs</title>
```
Comment (abhishekp2903) [inline index.html:9]: Leave title as it was earlier

**Review by abhishekp2903 — CHANGES_REQUESTED** [review]
Solve the issue that I have mentioned in the file comment and then merge it

File: index.html (hunk 1 above)
Comment (abhishekp2903) [reply index.html:9]: OK, fine

Comment (abhishekp2917) [reply index.html:9]: ok, fine. I'll do the same

**Pull request closed without merging**